driving-day-app-frontend/src/components/driver-components/SpecificDriverProfile.tsx

```diff
@@ -1,34 +1,21 @@
-// // SpecificDriverProfile.tsx
-// import React from "react";
-// import { useParams } from "react-router-dom";
+import React from "react";
+import { Driver } from "./Driver";
 
-// interface Driver {
-//     name: string;
-//     height: number;
-//     weight: number;
-// }
-
-// const defaultDriverData: Driver[] = [
-//     { name: "John Doe", height: 180, weight: 75 },
-//     { name: "Jane Smith", height: 165, weight: 60 },
-//     { name: "Mike Brown", height: 170, weight: 68 },
-//     { name: "Emily Davis", height: 158, weight: 55 },
-//     { name: "Chris Johnson", height: 185, weight: 85 },
-//     { name: "Sarah Wilson", height: 162, weight: 58 },
-// ];
-
-// export default function SpecificDriverProfile() {
-//     const { index } = useParams<{ index: string }>();
-//     const driver = defaultDriverData[parseInt(index || "0", 10)]; // Get driver by index
-
-//     if (!driver) return <div>Driver not found.</div>;
```

Comment:
I thought this was good to have before incase the user cannot find a driver. Maybe add it back in the current functionality in its respective location?